metricbeat/module/exec/doc.go

```diff
@@ -0,0 +1,6 @@
+/*
+
+Periodically executes command line scripts
```

Comment:
I suggest leaving this out until we have an implementation. Just in case we decide an external Execbeat would make sense on its own. I do start to believe that Metricbeat and Execbeat are different enough to be separate.